src/generator/exercises/templates/example.tt

```diff
@@ -3,7 +3,7 @@ require "../src/*"
 
 describe <%= "#{describe_name.inspect} do" %>
 <% test_cases.each_with_index do |test_case, index| %>
-  <%= test_case.pending?(index) %> "tests <%= test_case.test_name %>" do
```

Comment:
Ahhh derp... that's right. I completely forgot about running format before submitting.